src/shell/initrc.lua

```diff
@@ -0,0 +1,17 @@
+
+-- Load all *.so plugins from plugin.serarchpath
```

Comment:
Spelling: "searchpath"

It may be worth noting that this is the default in-tree initrc.lua in the comment at the top, and also in the commit message which says it's an example initrc.lua.

I may be missing it but is there intended to be an installed default initrc.lua also?

Reply:
> I may be missing it but is there intended to be an installed default initrc.lua also?

Yeah, that is not there yet, sorry. I meant to mention that this PR is up for early review to ensure the general approach is acceptable (before spending time writing tests on something that will have to change)

The example `initrc.lua` might be a good default, however I started to get a bit nervous about jobs reading from `~/.flux/shell/initrc.lua` by default, because a small error/typo in that script would cause a job to fail at runtime. It seemed like perhaps this approach might be setting users up for frustration.

Maybe it would be better if the user initrc was put directly into the jobspec (not exactly sure how that would work), allowing per-job customization instead of one global initrc for all of a user's jobs?

Anyway, I wasn't sure at this point and wanted to field a consensus before proceeding.

Reply:
> It may be worth noting that this is the default in-tree initrc.lua in the comment at the top, and also in the commit message which says it's an example initrc.lua.

Yeah, `src/shell/initrc.lua` is the default in-tree shell initrc. I'll update the confusing commit message. I'm actually a bit torn on what should be *in* the default in-tree initrc. Might not be wise or necessary to read from a user's custom initrc here!

Reply:
Ok fixed the spelling error and updated the commit message to be more explicit.